modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -204,11 +218,36 @@ AbaqusUMATStress::computeQpStress()
   const Real * myDFGRD1 = &(FBar_fortran(0, 0));
   const Real * myDROT = &(DROT_fortran(0, 0));
 
+  // More local copies of materials so we can rotate
```

Comment:
we don't need to make a copy if we don't rotate

Reply:
We do rotate these below

Reply:
Oh do you mean to skip this step if not using orientation? The rotation is optional but we need these quantities in [this section](https://github.com/jessecarterMOOSE/moose/blob/2b3635dab1eb1e044e3e2d665e77d0347da27f3b/modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C#L265-L272). If there is a cleaner way to do this then I'd be open to it.

Reply:
This is the cleanest way I can think of. The alternative would be have more conditionals, which also sacrifices readability.